Core/clim-core/presentation-translators.lisp

```diff
@@ -78,25 +78,19 @@ and used to ensure that presentation-translators-caches are up to date.")
                             :initform (make-hash-table :test #'eq)
                             :documentation "Holds transators with a simple
   from-type (i.e. doesn't contain \"or\" or \"and\").")
-   (translator-cache-generation :accessor translator-cache-generation
-                                :initform 0)
-   (presentation-translators-cache
-    :writer (setf presentation-translators-cache)
-    :initform (make-hash-table :test #'equal))))
+   (cache-gen :initform 0)
+   (cache :initform (make-hash-table :test #'equal))))
```

Comment:
I like `presentation-translators-cache` -> `cache` but I would prefer `cache-generation` for the other slot since "gen" can mean "generator" or "generation".

Reply:
done (and added accessors)